npm-publish/bin/publish.sh

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env bash
+
+set -o errexit   # exit on error
+set -o errtrace  # exit on error within function/sub-shell
+set -o nounset   # error on undefined vars
+set -o pipefail  # error if piped command fails
+
+# Default variables
+RELEASE_TYPE=
+MAIN_BRANCH="trunk"
```

Comment:
What do you think if we do something like this:

```suggestion
MAIN_BRANCH="$(LC_ALL=C git remote show origin | awk '/HEAD branch/ {print $NF}')"
```

This will work in case the default branch is not `trunk` but, say, `develop` or `main` or whatever (my personal preference is not to hardcode things if we can avoid that).

Reply:
Oh, good thinking! Much better.